sys/stop.g

```diff
@@ -3,4 +3,7 @@
 ; Called on cancelling or finishing a job.
 ; Apparently also triggered when pausing.
 ; Park the spindle.
-G27
\ No newline at end of file
+G27
+
+; Disable Coolant 
+M9
```

Comment:
There shouldn't be any need to call `M9` here because it's already called inside `G27`